pkg/controller/certificates/sync.go

```diff
@@ -135,6 +135,9 @@ func (c *certificateRequestManager) updateCertificateStatus(ctx context.Context,
 		ready = cmmeta.ConditionTrue
 		reason = "Ready"
 		message = "Certificate is up to date and has not expired"
+	case apiutil.CertificateRequestHasInvalidRequest(req):
+		reason = "InvalidRequest"
+		message = fmt.Sprintf("The CertificateRequest %q is an invalid request", req.Name)
```

Comment:
Can we provide some more useful actionable info here? `fmt.Sprintf("The certificate request could not be completed due to invalid request options: %s", req.Status.Conditions[InvalidRequestCondition].Message)`